functions/transformations/keys.go

```diff
@@ -162,42 +161,48 @@ func (t *keysTransformation) Process(id execute.DatasetID, tbl flux.Table) error
 		return fmt.Errorf("keys found duplicate table with key: %v", tbl.Key())
 	}
 
```

Comment:
I guess it's no longer necessary to sort the `except` array when creating the transformation above, here:
https://github.com/influxdata/flux/blob/3fc3eeaa76952f5d732ab17089d5de5249142fbe/functions/transformations/keys.go#L144

Reply:
Good catch. I'll update that.